GooglePayment/build.gradle

```diff
@@ -54,6 +60,14 @@ dependencies {
     testImplementation 'org.robolectric:robolectric:4.3'
     testImplementation 'org.skyscreamer:jsonassert:1.4.0'
     testImplementation 'com.squareup.assertj:assertj-android:1.1.1'
+
+    androidTestImplementation 'junit:junit:4.12'
+    androidTestImplementation 'androidx.test:runner:1.1.0'
+    androidTestImplementation 'androidx.test:rules:1.1.0'
+//    androidTestImplementation 'org.powermock:powermock-module-junit4:1.6.6'
```

Comment:
If we don't need these, can we remove them?

Reply:
Yep, good catch!